invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.variables

```diff
@@ -34,3 +34,8 @@
 
 @headerFont        : @fontName, sans-serif;
 @pageFont          : @fontName, sans-serif;
+
+/*--- Focus ---*/
+
+@focusColor : #3dbbdb;
+@focusBackgroundColor : inherit;
```

Comment:
I don't see the usage of this variable, am I missing something? If it is not used, it should be removed